### ISSUE TITLE
[Component] [form-item] [accessibility] Missing role on for item error message

### ISSUE THREAD

**Issue by chou666** [element-plus/element-plus#24673]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.5.41`
- Element Plus Version: `2.14.3`
- Browser / OS: `Firefox / Ubuntu`
- Build Tool: `Vite`

### Related Component

- `el-form-item`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/form#validation)

### Steps to reproduce

Click on create button

### What is Expected?

Fix: Added role="alert" and aria-live=‘polite’ to the error messages

### What is actually happening?

Nothing

### Additional comments

Error messages relating to incorrect or missing entries in fields are not displayed by the text assistant.
